### DISCUSSION TITLE
Payments

### DISCUSSION THREAD

**Discussion by LittleBitsSoftware** [swellstores/community#249] [Help] [answered]
I am working on a project where products are usually not shipped for a few months. Currently, payments are authorized at the time of purchase. How do I capture the payment when the items are shipped?

**Comment by sags95** [comment]
@RobTrainerLBS One way would be to disable the 'Charge Automatically' setting in Settings > Orders. This way the authorization step and capture can be done separately. You can then capture the payment manually through the dashboard, or through API when the product is shipped.

**Reply by LittleBitsSoftware** [reply]
@sags95 Thank you. What you are saying is that there isn't a way through the API to capture funds that were authorized but not captured at the time of the order.

**Reply by sags95** [reply, marked as answer]
There is a way to do so:

1. Order that was created will have a payment that is only authorized. You can query for the payment details to retrieve the payment `id` with:

```
await swell.get('/orders/<id>', {
 expand: ['payments']
}
```
2. Update the payment to capture the charge:

```
await swell.put('/payments/<payment_id>', {
   captured: true
}
```

The charge will then be captured.

**Reply by LittleBitsSoftware** [reply]
Awesome.  Thank you.

Where is this documented?

**Reply by sags95** [reply]
@RobTrainerLBS Looking over the Backend API documentation it seems that this is not mentioned, so I'll make a note to our documentation team to add this.
